src/components/ManagedDrop.tsx

```diff
@@ -7,21 +9,20 @@ type DropdownProps = {
 
 export function ManagedDrop (props: DropdownProps) {
     const yesNo = [
-        'None',
-        'Yes',
-        'No'
+        { value: 'None', label: 'None' },
+        { value: 'Yes', label: 'Yes' },
+        { value: 'No', label: 'No' }
     ]
 
     return (
-        <label className="seven dropdown"> Managed <span className='downArrow'> ⬇ </span>
-            <p className="smaller-font">Defaults to None</p>
-            <select multiple value={props.managed} className="dropdown-content" onChange={(e: ChangeEvent<HTMLSelectElement>) => {
-                props.changedManaged([...e.target.selectedOptions].map((option) => option.value))
-            }} tabIndex={3} >
-                {yesNo.map((isManaged) => {
-                    return <option key={isManaged} value={isManaged}>{isManaged}</option>
-                })}
-            </select>
+        <label className="seven"> Managed
+            <Select
+                defaultValue={yesNo[0]}
+                getValue={props.managed}
+                onChange={(e: PropertyTypes) => props.changedManaged([e.value])}
```

Comment:
@haydenf This is the part that I changed. You were using `setValue` when it needed to be `onChange`.

Also, the event wasn't the standard form event b/c it went through this package it came out as `{value: string, label: string}`. So you didn't have to map through, but just had to access the value key.